_bmad-output/implementation-artifacts/stories/3-3-safe-subscriber-unregistration.md

```
@@ -0,0 +1,300 @@
+        ROLLBACK;
+        RAISE;
+    END IF;
+END Register_Subscriber;
+```
+
+**Unregister_Subscriber to implement** (mirror of Register, using REMOVE_SUBSCRIBER):
+- Oracle error for subscriber not found: ORA-24036 (SQLCODE = -24036)
+- Must be `PRAGMA AUTONOMOUS_TRANSACTION` (same as Register)
+
+**5-second timeout pattern**:
+```go
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)